doc/missing-references.json

```diff
@@ -195,6 +200,70 @@
     "dviread.PsfontsMap": [
       "lib/matplotlib/dviread.py:docstring of matplotlib.dviread.DviFont:20"
     ],
+    "matplotlib._mathtext.Box": [
+      "doc/api/mathtext_api.rst:11",
```

Comment:
It makes more sense to update these references, or delete them, rather than start accepting them as missing.

Reply:
The problem is that (AFAICT) at least they will always show up in the SVG inheritance diagram.  I got rid of the other occurrences, though.